motionplan/motionPlanner.go

```diff
@@ -54,10 +54,49 @@ type PlanRequest struct {
 	Options            map[string]interface{}
 }
 
+// PlanRequestValidation ensures PlanRequests are not malformed.
+func PlanRequestValidation(req PlanRequest) (*PlanRequest, error) {
+	if req.Logger == nil {
+		req.Logger = golog.NewLogger("plan-request-logger")
+		req.Logger.Info("no logger was given, so we created one")
+	}
+	if req.Goal == nil {
+		return &req, errors.New("PlanRequest cannot have nil goal")
+	}
+	if req.Frame == nil {
+		return &req, errors.New("PlanRequest cannot have nil frame")
+	}
+	if req.FrameSystem == nil {
+		req.Logger.Info("no FrameSystem was given, so we ephemerally create a framesystem containing just the frame for the solve")
```

Comment:
This should be in `Debug`, otherwise this will hit the log every time anyone does any arm movement whatsoever.

Reply:
I don't think we should be logging this

Reply:
I think it should remain. I am imagining where we succeed in producing a plan, but the user did not specify a frame system, in this case i think it would serve us well to be more informative.

In the case where we do not succeed in producing a plan, I also think this would be good information to have with debugging.

Reply:
I'm okay with logging it as long as it's in debug; considering that this isn't a failing condition, but will produce different behavior in the plan, if something that ought be producing a framesystem was failing to do so then this would catch that.